### ISSUE TITLE
Intercom - Log Event

### ISSUE THREAD

**Issue by sasquatchjordan** [PipedreamHQ/pipedream#4574]
**Name of app / service**
Intercom

**Link to developer documentation**
https://developers.intercom.com/intercom-api-reference/reference/submitting-events

**Is lack of support preventing you from building workflows, or do you have a workaround?**
I haven't yet... I could try to build something myself

**Are there specific actions, or triggers, you'd like to see for this app? Please let us know here or use the Action and Trigger issue templates to open requests for each!**
Basically, need the ability to look up a user and add an event to their account. This can then kick-off actions and emails inside intercom. I think Zapier has this here:
"Add Event
Tags a user with an event that occurred."

**Comment by sergioeliot2039** [comment]
Thanks for your request @sasquatchjordan !!

I looks like that trigger is already integrated, can you please check if it works for you?

https://pipedream.com/apps/intercom/triggers/new-event

> **New Event from the Intercom API:**
> Emit new event for each new Intercom event for a user

![image](https://user-images.githubusercontent.com/25184117/197632518-a0d59a50-9bf3-4477-88d1-107d95e3bcce.png)

Best,

Sergio

**Comment by dylburger** [comment]
Without more information, I'm going to close this issue. Feel free to reopen and add more details, if you'd like.
